app/assets/stylesheets/components/headers.scss

```diff
@@ -33,7 +41,4 @@
 
 .crayons-subtitle-3 {
   font-size: var(--fs-base);
-  @media (min-width: $breakpoint-s) {
-    font-size: var(--fs-l);
-  }
```

Comment:
Why this? It's currently used not only in Mod but also notifications. Making the headline a little bit bigger on desktop sizes looks OK imo.. What's the rationale here?

Reply:
Since the subtitle is already bold, I think that there's enough contrast between the subtitle and its the nav link blocks, and it doesn't seem necessary to create more differentiation. I think that with a `fs-base`, the subtitle and the set of nav links hold tighter together.

By the way, in my initial CSS proposal, I intended for the size in "Notifications" to also use `fs-base` as well for desktop for the same reasons.

Here is a side-by-side screenshot:

<img width="786" alt="Screen Shot 2020-10-06 at 7 12 00 PM" src="https://user-images.githubusercontent.com/816402/95279616-044f5280-0808-11eb-832b-c77a54e6e1ec.png">